### PR TITLE
Introduce 100LinesOfCode

### PR DESCRIPTION
**Pull request by josharsh**

### CODE REVIEW

**Review by ritwik12 — CHANGES_REQUESTED** [review]
@josharsh You have a bad rebase due to which commits are mixed up. Please check
Also, change the name of the PR. why are you having your name as PR name?

File: _data/projects/100LinesOfCode.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: 100-Lines-Of-Code
+desc: "This repository contains productive coded tools in less than #100LinesOfCode."
+site: https://github.com/josharsh/100LinesOfCode
+tags:
+- josharsh
```
Comment (ritwik12) [inline _data/projects/100LinesOfCode.yml]: @josharsh Thats your username, why is it a Tag?

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/100LinesOfCode.yml (hunk 1 above)
Comment (josharsh) [reply _data/projects/100LinesOfCode.yml]: corrected!

**Comment by ritwik12** [comment]
@josharsh Thanks!!

**Comment by ritwik12** [comment]
@josharsh  See that's what I was telling you Above.. you had a bad pull and rebase due to which other commits were also shown in your PR. Even after my squash and merge it still shows two contributors for your PR which is wrong. This screws the commit history and tree. better to clean your commits in future when you do so. I'm telling just for your knowledge and learning :)

**Pull request merged**